requirements.sweeps.txt

```
@@ -1,7 +1,7 @@
-numpy>=1.15,<1.21
+numpy>=1.15
 scipy>=1.5.4
 PyYAML
-scikit_learn==0.24.1
+scikit_learn==0.24.1; sys_platform != 'darwin' or (sys_platform == 'darwin' and platform.machine != 'arm64')
```

```suggestion
scikit_learn==0.24.1
```

i think we should just leave this as is